### DISCUSSION TITLE
Only mouse tap sounds are positional, while other UI sounds are not

### DISCUSSION THREAD

**Discussion by Susko3** [ppy/osu#16804] [Q&A]
Feels weird that the cursor tap sound is [panned to the left or right](https://github.com/ppy/osu/blob/885a285d13c0469c1e8de94b56cc0514396adeaa/osu.Game/Graphics/Cursor/MenuCursor.cs#L140-L141), while other UI sounds (hover, select) are not.

**Comment by peppy** [comment]
It does seem like we want any UI interaction sounds to be balanced to the mouse cursor. For something like select it is a bit more complicated as it may be triggered via keyboard or controller.

Probably want to wait until we figure out the audio mixer stack before making any changes here, but is something worth considering in the future.
